addons/mass_mailing_sale/models/mailing_mailing.py

```diff
@@ -15,47 +15,46 @@ class MassMailing(models.Model):
 
     @api.depends('mailing_domain')
     def _compute_sale_quotation_count(self):
-        has_so_access = self.env['sale.order'].check_access_rights('read', raise_exception=False)
-        if not has_so_access:
-            self.sale_quotation_count = 0
-            return
         for mass_mailing in self:
-            mass_mailing.sale_quotation_count = self.env['sale.order'].search_count(mass_mailing._get_sale_utm_domain())
+            mass_mailing.sale_quotation_count = self.env['sale.order'].sudo().search_count(mass_mailing._get_sale_utm_domain())
 
     @api.depends('mailing_domain')
     def _compute_sale_invoiced_amount(self):
-        if not self.user_has_groups('sales_team.group_sale_salesman') or not self.user_has_groups('account.group_account_invoice'):
-            self.sale_invoiced_amount = 0
-            return
         for mass_mailing in self:
             domain = expression.AND([
                 mass_mailing._get_sale_utm_domain(),
                 [('state', 'not in', ['draft', 'cancel'])]
             ])
-            moves = self.env['account.move'].search_read(domain, ['amount_untaxed_signed'])
+            moves = self.env['account.move'].sudo().search_read(domain, ['amount_untaxed_signed'])
```

Comment:
Same here, proposal to use only the source and get rid of sale utm domain usage (complex for few real use cases I think)